e2e/resolve-with-paths/__tests__/resolveWithPaths.test.js

```diff
@@ -35,3 +35,9 @@ test('throws an error if the module cannot be found from given paths', () => {
     "Cannot resolve module './mod.js' from paths ['..'] from "
   );
 });
+
+test('throws module not found error if the module cannot be found from given paths', () => {
+  expect(() =>
+    require.resolve('./mod.js', {paths: ['..']})
+  ).rejects.toMatchObject({code: 'MODULE_NOT_FOUND'});
```

Comment:
Oh sorry, this isn't async.

Probably

```js
).toThrow(expect.objectMatching({code: 'MODULE_NOT_FOUND'}));
```

My bad

(Written on mobile, please excuse typos)

Reply:
Thanks